src/pages/borrowers.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from 'react';
+import Helmet from 'react-helmet';
+import { HeaderComponent } from '../components/HeaderComponent';
+import fav from '../../static/favicon-32x32.png'
+import preview from '../../static/preview_large.png'
+import { HomeComponent } from '../components/borrowers/HomeComponent';
+import { FooterComponent } from '../components/FooterComponent';
+import { CTAComponent } from '../components/borrowers/CTAComponent';
+import { CTARegisterComponent } from '../components/borrowers/CTARegisterComponent';
+import { InvestorsList } from '../components/investors/InvestorsList';
+import { Quotes } from '../components/borrowers/Quotes';
+
+
+const BorrowersPage = () => {
+  const [width, setWidth] = useState<number>();
```

Comment:
actually if you want you can create a custom hook for this since it's used in multiple pages

Reply:
I mean for the entire part: 

```
 const [width, setWidth] = useState<number>();
  const mobileWidth = 900;
  const tabletWidth = 1500;

  useEffect(() => {
    setWidth(window.innerWidth);
    window.addEventListener('resize', () => {
      setWidth(window.innerWidth);
    });
    return () => {
      window.removeEventListener('resize', () => {return;});
    };
  }, []);
```

Reply:
done